Compiling for Mingw fails (Audio Driver)

**Godot version:**
master

**OS/device including version:**
Antergos OS (Arch Linux)


**Issue description:**
Compiling for Mingw fails after recent pull (maybe https://github.com/godotengine/godot/commit/1570a72eee4bc0efb46cc2185c44a12a0aa57943 ?)


**Steps to reproduce:**
scons p=windows bits=64 tools=no target=release -j 8

**Error Output:**
```Swift
[ 72%] Compiling ==> drivers/gles3/shader_compiler_gles3.cpp
[ 72%] Linking Static Library ==> modules/libmodules.windows.opt.debug.64.a
drivers/wasapi/audio_driver_wasapi.cpp: In member function 'Error AudioDriverWASAPI::init_device(bool)':
drivers/wasapi/audio_driver_wasapi.cpp:154:25: error: 'PKEY_Device_FriendlyName' was not declared in this scope
    hr = props->GetValue(PKEY_Device_FriendlyName, &propvar);
                         ^~~~~~~~~~~~~~~~~~~~~~~~
drivers/wasapi/audio_driver_wasapi.cpp: In member function 'virtual Array AudioDriverWASAPI::get_device_list()':
drivers/wasapi/audio_driver_wasapi.cpp:386:24: error: 'PKEY_Device_FriendlyName' was not declared in this scope
   hr = props->GetValue(PKEY_Device_FriendlyName, &propvar);
                        ^~~~~~~~~~~~~~~~~~~~~~~~
[ 73%] Compiling ==> drivers/gles3/rasterizer_scene_gles3.cpp
[ 73%] Compiling ==> drivers/gles2/rasterizer_canvas_gles2.cpp
scons: *** [drivers/wasapi/audio_driver_wasapi.windows.opt.debug.64.o] Error 1
Ranlib Library         ==> modules/libmodules.windows.opt.debug.64.a
scons: building terminated because of errors.
```

--- a/drivers/wasapi/audio_driver_wasapi.cpp
+++ b/drivers/wasapi/audio_driver_wasapi.cpp
@@ -37,6 +37,17 @@
 
 #include <functiondiscoverykeys.h>
 
+#ifndef PKEY_Device_FriendlyName
+
+#undef DEFINE_PROPERTYKEY
+/* clang-format off */
+#define DEFINE_PROPERTYKEY(id, a, b, c, d, e, f, g, h, i, j, k, l) \
+	const PROPERTYKEY id = { { a, b, c, { d, e, f, g, h, i, j, k, } }, l };
+/* clang-format on */
+
+DEFINE_PROPERTYKEY(PKEY_Device_FriendlyName, 0xa45c254e, 0xdf1c, 0x4efd, 0x80, 0x20, 0x67, 0xd1, 0x46, 0xa8, 0x50, 0xe0, 14);
+#endif
+
 const CLSID CLSID_MMDeviceEnumerator = __uuidof(MMDeviceEnumerator);
 const IID IID_IMMDeviceEnumerator = __uuidof(IMMDeviceEnumerator);
 const IID IID_IAudioClient = __uuidof(IAudioClient);

